src/OrbitGl/App.cpp

```diff
@@ -2269,6 +2313,24 @@ orbit_base::Future<ErrorMessageOr<void>> OrbitApp::UpdateProcessAndModuleList()
       });
 }
 
+Future<std::vector<ErrorMessageOr<void>>> OrbitApp::LoadAllSymbols() {
+  const ProcessData& process = GetConnectedOrLoadedProcess();
+
+  std::vector<const ModuleData*> sorted_module_list = SortModuleListWithPrioritizationList(
+      module_manager_->GetAllModuleData(),
+      std::vector<std::string_view>{kGgpVlkModulePathSubstring, process.full_path()});
```

Comment:
There should be no need to wrap this into a vector. `{kGgpVlkModulePathSubstring, process.full_path()}` should work. Maybe you need a cast.